src/components/Dialog.js

```diff
@@ -1,6 +1,29 @@
 import { CURRENT_WINDOW } from './Window';
 import libui from 'libui-node';
 
+const getCorrectDialog = {
```

Comment:
I think there's a typo here. You define it as `getCorrectDialog`
But call it as `getCurrentDialog[type](options)` later

Reply:
Oh good catch! Fixed on my last commit!